docsets/Vaadin/docset.json

```diff
@@ -1,6 +1,6 @@
-{
+ate{
```

Comment:
Typo?

Reply:
There were some weird errors in the doc conversion, but oddly it still worked okay for me?

In any case, we have a new version now so I'll close out this one.